isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -347,54 +347,24 @@
         </filter>
       </parameter>
 
- <!-- Turning off USEPVL and SC_PARAMETERS for now. Since we are now using observation numbers
-      to obtain the BundleObservationSolveSettings for a BundleObservation (instead of inst. id),
-      this would require using a PVL that contains names of lists of images to group by.
-      E.g. Group = "listA." would create a BundleObservationSolveSettings object that is associated
-      with the images' observation numbers. Group = "listB.lis" would create another BOSS object
-      that is associated with listB's images' observation numbers. Use of these lists could
-      introduce unintended behaviors if not properly verified (e.g. would need to make sure the
-      lists are sublists of the FROMLIST, that these sublists are all mutually exclusive ...).
-      <parameter name="USEPVL">
-        <type>boolean</type>
-        <default><item>false</item></default>
-        <brief>
-          Enter Camera/Spacecraft parameters using a PVL file
-        </brief>
-        <description>
-          This option indicates that the Camera and Spacecraft parameters are to be
-          obtained from the PVL file specified by SC_PARAMETERS. All of the following
-          parameters must be entered through the PVL file if this option is chosen:
-          HELDLIST, CKDEGREE, CKSOLVEDEGREE, CAMSOLVE, TWIST, OVEREXISTING, SPKDEGREE,
-          SPKSOLVEDEGREE, SPSOLVE, OVERHERMITE, SPACECRAFT_POSITION_SIGMA,
-          SPACECRAFT_VELOCITY_SIGMA, SPACECRAFT_ACCELERATION_SIGMA, CAMERA_ANGLES_SIGMA,
-          CAMERA_ANGULAR_VELOCITY_SIGMA, CAMERA_ANGULAR_ACCELERATION_SIGMA. An example
-          template is located at $base/templates/jigsaw/SC_Parameters.pvl.
-        </description>
+      <parameter name="SCCONFIG">
         <exclusions>
```

Comment:
currently, I've only added parameters that the BOSS constructor populates into the exclusion list.
I'm sure there are other parameters that should be excluded, but I have a hard time figuring out which ones.